src/lib/components/ProductForm.svelte

```diff
@@ -652,19 +691,39 @@
 							/>
 						</label>
 						<label class="form-label">
-							Value <input
+							{i === 0 ? 'Value' : ''}
+							<input
 								type="text"
 								name="variationLabels.values[{variation.name}][{variation.value}]"
 								class="form-input"
 								value={product.variationLabels?.values[variation.name]?.[variation.value]}
 								bind:this={variationInput[i]}
-								on:input={() => variationInput[i]?.setCustomValidity('')}
+								on:input={() => (
+									variationInput[i]?.setCustomValidity(''), (variationInput[i] = variationInput[i])
+								)}
```

Comment:
First, why does this return a tuple instead of using a regular function body?

```suggestion
								on:input={() => {
									variationInput[i]?.setCustomValidity('');
									variationInput[i] = variationInput[i];
								}}
```

Second, what's the purpose of `variationInput[i] = variationInput[i]`?